source/parser/x86_64/classifiers.hpp

```diff
@@ -74,7 +74,6 @@ namespace smeagle::x86_64 {
         }
         // This is NOT correct.
         // TODO It should be struct{T r,i;};, but we don't handle aggregates yet
```

Comment:
Is this comment still the case?

Reply:
Yup. It's something we need to work on. It's a toss-up between that and enums as the next step. I need to get references and constants working in Dyninst before we can add them to Smeagle.